modules/programs/k9s.nix

```
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.k9s;
+  yamlFormat = pkgs.formats.yaml { };
+in {
+  meta.maintainers = [ hm.maintainers.katexochen ];
+
+  options.programs.k9s = {
+    enable =
+      mkEnableOption "k9s - Kubernetes CLI To Manage Your Clusters In Style!";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.k9s;
+      defaultText = literalExpression "pkgs.k9s";
+      description = "The k9s package to install.";
+    };
```

Can use the new `mkPackageOption` function:

```suggestion
    package = mkPackageOption pkgs "k9s { };
```